cpp/include/cuspatial/cubic_spline.hpp

```diff
@@ -19,6 +19,7 @@
 #include <cudf/column/column.hpp>
 #include <cudf/table/table.hpp>
 #include <memory>
+#include "cudf/types.hpp"
```

Comment:
Should this be bracketed instead of quoted?

Reply:
Yes I think so